src/coreclr/jit/lsra.cpp

```diff
@@ -507,6 +507,46 @@ regMaskTP LinearScan::getConstrainedRegMask(RefPosition* refPosition,
     return newMask;
 }
 
+//------------------------------------------------------------------------
```

Comment:
This is fixing build break in Linux debug builds (looks like a bug in the C++ compiler/linker to me):

https://dnceng-public.visualstudio.com/public/_build/results?buildId=672464&view=logs&j=689957d3-95e5-570c-99ed-eaa47e031b3f&t=a66b4246-3d14-52ac-fc42-6b6b8bdf0ccd&l=3374